js/app.js

```diff
@@ -26,86 +26,68 @@ $(document).ready(function () {
 	const defaultLineHeight = 26;
 	const defaultFontWeight = 'normal';
 	const defaultShowWordCountPill = 'Yes';
+	const { notepad: _, state, setState, removeState, get } = selector();
 
-	if (localStorage.getItem('note') && localStorage.getItem('note') != '') {
-		const noteItem = localStorage.getItem('note');
-
+	if (state.note && state.note != '') {
+		const noteItem = state.note;
 		const characterAndWordCountText = calculateCharactersAndWords(noteItem);
-
-		$('#wordCount').text(characterAndWordCountText);
-
-		$('#note').val(noteItem);
+		_.wordCount.text(characterAndWordCountText);
+		_.note.val(noteItem);
```

Comment:
can be written i.e **notepad.note.val(noteItem)**

Reply:
I like this more.